ui/pages/providers/add/details.js

```diff
@@ -167,6 +177,67 @@ export default function ProvidersAddDetails() {
             <ErrorMessage message={errors.clientsecret} />
           )}
         </div>
+        {kind === 'google' && (
+          <div>
+            <label className='text-2xs text-white/90'>
+              Optional details{' '}
+              <a
+                className='text-violet-100 underline'
+                target='_blank'
+                href='https://infrahq.com/docs/identity-providers/google#groups' /* TODO: make sure this link works*/
+                rel='noreferrer'
+              >
+                learn more
+              </a>
+            </label>
+            <div className='mt-4'>
+              <label className='text-3xs uppercase text-gray-400'>
+                Private Key
+              </label>
+              <input
+                type='password'
+                placeholder='service account private key'
+                value={privateKey}
+                onChange={e => setPrivateKey(e.target.value)}
+                className={`w-full border-b border-gray-800 bg-transparent px-px py-3 text-3xs placeholder:italic focus:border-b focus:border-gray-200 focus:outline-none ${
+                  errors.url ? 'border-pink-500/60' : ''
+                }`}
+              />
+              {errors.url && <ErrorMessage message={errors.url} />}
+            </div>
+            <div className='mt-4'>
+              <label className='text-3xs uppercase text-gray-400'>
+                Client Email
+              </label>
+              <input
+                placeholder='client email'
+                type='search'
```

Comment:
do we have validation on this field? Maybe we can change the type to `email`? and it should be ok to trigger the 1password on the email field